NuRadioReco/framework/event.py

```diff
@@ -376,8 +382,9 @@ def get_sim_showers(self):
         """
         Get an iterator over all simulated showers in the event
         """
-        for shower in self.__sim_showers.values():
-            yield shower
+        return self.__sim_showers.values()
+        # for shower in self.__sim_showers.values():
+        #     yield shower
```

Comment:
change the doc-string. Typically this kind of functions returns an iterator right? Why changing that for this one?

Reply:
Ping - wrong doc-string